openstates/az/bills.py

```diff
@@ -258,7 +258,7 @@ def scrape_votes(self, bill, page):
                         'N': 'no',
                     }.get(v['Vote'], 'other')
                     vote.vote(vote_type, v['Legislator']['FullName'])
-
+                vote.pupa_id = resp.url+str(action['ReferralNumber'])
```

Comment:
I was wondering if we could add the referral number to `extras` to deal with apparent duplicates. That's one of the issues I was thinking about in https://github.com/opencivicdata/python-opencivicdata/pull/118. Would that work? Is it better to add extra fields or to set a custom pupa id? cc @jamesturk